AdyenActions/Components/Await/AwaitComponent.swift

```diff
@@ -22,35 +22,53 @@ public final class AwaitComponent: ActionComponent, Cancellable {
     /// :nodoc:
     public let requiresModalPresentation: Bool = true
     
-    /// The Component UI style.
-    public let style: AwaitComponentStyle
+    /// The await component configurations.
+    public struct Configuration {
```

Comment:
I think all of them would be better conforming to `AnyBasicComponentConfiguration` for consistency.